# Special initial positions so that osc isnt at singularity

initial_positions:
  fr3_joint_linear: 0.0
  fr3_joint1: 0.0
  fr3_joint2: -0.3949
  fr3_joint3: 0.0
  fr3_joint4: -2.0177
  fr3_joint5: 0.0
  fr3_joint6: 1.5543
  fr3_joint7: 0.7377
  fr3_finger_joint1: 0